netCDF4.pyx

```diff
@@ -2465,6 +2465,8 @@ instance. If C{None}, the data is not truncated. """
         cdef size_t sizep, nelemsp
         cdef size_t *chunksizesp
         cdef float preemptionp
+        # flag to indicate that orthogonal indexing is supported
+        self.__orthogonal_indexing__ = True
```

Comment:
This could be a class attribute instead of an instance attribute, e.g., you could just set `__orthogonal_indexing__ = True` at one higher level.